docs/pages/en-US/marketplace/halo.mdx

```diff
@@ -1,78 +1,81 @@
 ---
 title: Deploy Halo
 ogImageTitle: Deploy Halo
-ogImageSubtitle: Deploy Halo on Zeabur
+ogImageSubtitle: One-click Deploy Halo Service on Zeabur
 ---
 
 import { Callout } from 'nextra/components'
 
-# Deploy Halo Service
+# Deploy Halo
 
-[Halo](https://halo.run/) is a powerful and easy-to-use open source website building tool, with rich themes and plugins to help you build the ideal site in your mind.
+[Halo](https://www.halo.run/) is a powerful and easy-to-use open-source website building tool. With a rich set of templates and plugins, it helps you build your ideal website.
 
-This article will teach you how to deploy your own Halo service on Zeabur through the service market, and bind your own domain name.
+This article will guide you on how to easily deploy your own Halo service on Zeabur through the service marketplace and bind your own domain name.
 
-## Step 1: Create a project
+## Deploy Halo from Template
```

Comment:
This subtitle may be unnecessary. Perhaps you can incorporate the steps into Title 2 (`##`) instead.

Reply:
Updated